src/Features/Core/Portable/SolutionCrawler/ISolutionCrawlerProgressReporter.cs

```diff
@@ -15,13 +15,10 @@ internal interface ISolutionCrawlerProgressReporter
         bool InProgress { get; }
 
         /// <summary>
-        /// Raised when there is pending work in solution crawler.
+        /// Raised when solution crawler progress changed
+        /// 
+        /// bool indicates whether progress is started or stopped
```

Comment:
:bulb:

```csharp
/// The event handler argument is <see langword="true"/> when the solution crawler is running; otherwise,
/// <see langword="false"/> to indicate that the crawler has stopped.
```

:bulb: If this interface is used by downstream consumers (e.g. TypeScript, F#), consider changing `bool` to some value type which we can modify in the future (e.g. to include progress) without it being a breaking change. If this type is only used within Roslyn, there is no need to change it now since we can simply change it when necessary.

Reply:
they have IVT so no idea what API they consume from us. but from the fact that they use Document/ProjectDiagnosticAnalyzer which means they use our Diagnostic Framework, they shouldn't care this since that is the reason they use our Diagnostic Framework so that they don't need to care about these kind of stuff.